doc/pct_format.md

```diff
@@ -0,0 +1,17 @@
+# PCT format
+
+PCT uses its own data format implemented as MHD image files in which data are organized in a particular way. The data corresponds to proton pairs, for which some information is stored, such as position, direction, and energy. The first proton represents the proton detected by the entrance detector, while the second proton represents the proton detected by the exit detector.
```

Comment:
MHD -> MetaImage (mhd or mha extention) with a link to https://itk.org/Wiki/ITK/MetaIO/Documentation.